### ISSUE TITLE
Add the ability to override the default database settings

### ISSUE THREAD

**Issue by TravisCarden** [phenaproxima/starshot-prototype#10]
Drupal Suite uses an SQLite database by default, which is most convenience for quick stand-up for local development, but unlikely to be used in production. Per https://github.com/phenaproxima/drupal-suite/pull/9#issuecomment-2070096704, provide the ability to override the default during `composer create-project` and use MySQL or Postgres instead. Document the feature.

**Comment by phenaproxima** [comment]
As of 20eeb2cb3c4859f6d7f7a70ac39079c5268d16ab, I did this in a somewhat hacky way, but it works.

Normally you'd be able to override the database credentials using an environment variable, which would look like this:

```
DRUSH_COMMAND_SITE_INSTALL_OPTIONS_DB_URL=$YOUR_DESIRED_DSN composer create-project drupal/starshot-project ...
```

However, due to a bug in Drush (addressed in drush-ops/drush#5961, which is not yet merged), that doesn't work. So here's the workaround:

```
composer create-project drupal/starshot-project --no-install ...
cd starshot-project
composer install
vendor/bin/yaml-cli update:value drush/drush.yml command.site.install.options.db-url $YOUR_DESIRED_DSN
composer drupal:install
```

Once the fix is merged into Drush, and in a tagged release, I'll be updating our Drush requirement so that you can use the environment variable (and I'll document that in the README).

**Comment by AlexSkrypnyk** [comment]
The Drush issue was merged, but will only be available in Drush 13. This project uses Drush 12.

**Comment by phenaproxima** [comment]
We _sorta_ have this as of #108:

- If you use DDEV, change the database backend you want to use (via the standard `ddev config --database=...` commands) before running `ddev install`.

**Comment by phenaproxima** [comment]
OK, we now fully have this as of #111.

If you're using DDEV, do what I said in my previous comment. If you're not, then you can pass a database URL (i.e., the `--db-url` option for `drush site:install`) to the `composer quick-start` command, like so:

```
composer quick-start mysql://foo:bar@localhost/starshot
```

All we still need to do is document it.

**Comment by phenaproxima** [comment]
Documented: https://github.com/phenaproxima/starshot-prototype/wiki#how-do-i-install-starshot-into-a-different-database
